Library/Formula/artifactory-cli-go.rb

```diff
@@ -0,0 +1,23 @@
+class ArtifactoryCliGo < Formula
+  desc "Artifactory-CLI provides a command-line interface."
+  homepage "https://github.com/JFrogDev/artifactory-cli-go"
+  url "https://github.com/JFrogDev/artifactory-cli-go/archive/1.2.0.tar.gz"
+  sha256 "173a24e0b821c6ff2bb85fa4cbe733fa681a8284029b77387baf0fa5f2fcfc4e"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOBIN"] = buildpath
+    ENV["GO15VENDOREXPERIMENT"] = "1"
+    (buildpath/"src/github.com/JFrogDev/artifactory-cli-go/").install Dir["*"]
+    system "go", "build", "-v", "github.com/JFrogDev/artifactory-cli-go/art/"
+    bin.install "art"
```

Comment:
can you help me in making this change. I am not getting what exactly I have to change here.